orc8r/gateway/python/magma/eventd/event_validator.py

```diff
@@ -0,0 +1,90 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json
+import logging
+from contextlib import closing
+from typing import Any, Dict
+import pkg_resources
+import yaml
+
+from bravado_core.spec import Spec
+from bravado_core.validate import validate_object as bravado_validate
+
+class EventValidator(object):
+    """
+    gRPC based server for EventD.
```

Comment:
Update docstring

Reply:
whoops auto merge :D will fix in following PRs